ops/terraform/env/mgmt-test/stateful/main.tf

```diff
@@ -0,0 +1,18 @@
+terraform {
+  required_version = "~> 0.12"
+}
+
+provider "aws" {
+  version = "~> 2.25"
+  region = "us-east-1"
+}
+
+module "stateful" {
+  source = "../../../modules/mgmt_stateful"
+
+  env_config = {
+    env               = "mgmt-test"
+    tags              = {application="bfd", business="oeda", stack="mgmt-test", Environment="mgmt-test"}
+    azs               = "us-east-1c"
```

Comment:
For standardization across modules I would move the `azs` var outside of env_config.

Reply:
This is how other modules are set for our other environments. Leaving this as is for now.